core/src/display_object.rs

```diff
@@ -703,6 +706,7 @@ pub fn render_base<'gc>(this: DisplayObject<'gc>, context: &mut RenderContext<'_
         let mut cache_info: Option<(BitmapHandle, bool, Transform, Rectangle<Twips>)> = None;
         let base_transform = context.transform_stack.transform();
         let bounds: Rectangle<Twips> = this.bounds_with_transform(&base_transform.matrix);
+        let path = this.path();
```

Comment:
I'm a little worried about the cost of calling `path` for every render, but it's true for `bounds_with_transform` too. I'll fix them both up in a followup PR